tracer/test/test-applications/integrations/Samples.Couchbase/Program.cs

```diff
@@ -15,6 +16,17 @@ internal class Program
         ICluster _cluster;
         IBucket _bucket;
 
+        private static bool ContainsAuthenticationException(Exception ex)
+        {
+            if (ex is AuthenticationException)
+                return true;
+
+            if (ex is AggregateException aggEx)
+                return aggEx.InnerExceptions.Any(ContainsAuthenticationException);
+
+            return ex.InnerException != null && ContainsAuthenticationException(ex.InnerException);
```

Comment:
nit: we use braces in general, but switch expressions make this terser anyway I think 😄  
```suggestion
        private static bool ContainsAuthenticationException(Exception ex) => ex switch
        {
            AuthenticationException => true,
            AggregateException aggEx => aggEx.InnerExceptions.Any(ContainsAuthenticationException),
            { InnerException: { } inner } => ContainsAuthenticationException(inner),
            _ => false,
        }
```

Reply:
Thanks!